### PR TITLE
phase1: make IsNoMasterBuild function more flexible

### PR DESCRIPTION
**Pull request by aparcar**
Work with both master/main

### CODE REVIEW

File: phase1/master.cfg [hunk 1]
```diff
@@ -562,7 +562,7 @@ c["schedulers"].append(
 
 
 def IsNoMasterBuild(step):
-    return step.getProperty("branch") != "main"
+    return step.getProperty("branch") != "main" and step.getProperty("branch") != "master"
```
Comment (ynezz) [inline phase1/master.cfg:565]:
```suggestion
    branch = step.getProperty("branch")
    return branch not in ['main', 'master']
```

nitpick, looks more idiomatic/readable?

**Review by ynezz — APPROVED** [review]
LGTM, thanks

File: phase1/master.cfg (hunk 1 above)
Comment (aparcar) [reply phase1/master.cfg:565]: Sure please go ahead

Comment (ynezz) [reply phase1/master.cfg:565]: You're signing commits, feel free to merge it yourself so the signature is kept.

**Comment by aparcar** [comment]
Done, thanks.

**Pull request merged**